Day3/application.py

```diff
@@ -27,16 +35,18 @@ def register():
       pwdcheck = request.form['psw-repeat']
       print(user,pwd,pwdcheck)
       # return render_template('/Register.html')
+      data = session.query(User).filter_by(user_id=user).all()
+      if(len(data) > 0):
+      	return "User already exists"
       if(pwd == pwdcheck):
          try:
             usr = User(user_id = user, pwd = pwd, timestamp = datetime.now())
             session.add(usr)
             session.commit()
-            return redirect(url_for('success',name = user, pwd = pwd))
+            return redirect(url_for('success',name = user))
```

Comment:
why not directly render the success page here instead of redirecting?

Reply:
That was to re-route Sir, is it okay to render the template are /register URL itself?